### ISSUE TITLE
Bootstrap v4 support?

### ISSUE THREAD

**Issue by WindSaber** [mouse0270/bootstrap-notify#183]
Are there any plans for bootstrap 4 support?

**Comment by baxxos** [comment]
It seems to run with BS4 beta-2 nicely for me, however it would be nice to know that it's officially supported.

**Comment by Kiruwagaka** [comment]
Any news on this particular issue?

**Comment by WindSaber** [comment]
@rolandoisidoro  You're right,  I had a mistake, the repo I wanted to share is https://jaspero.co/resources/projects/ng-notifications

**Comment by mouse0270** [comment]
This should work with any version of bootstrap by simply changing the template to include the new class names.

I will look into this in more detail since I now use bootstrap 4 too. 

Also sorry for the delay. I took a year-long break because I was working in a different field that used up way more of my free time than expected.
